pyproject.toml

```diff
@@ -66,5 +66,10 @@ dev = [
 profile = "google"
 line_length = 100
 
-[flake8]
-max_line_length = 100
+[tool.pylint.main]
+disable = [
+    "C0114",  # Missing module docstring
+    "E0401",  # Unable to import
+    "R0401",  # Cyclic import
```

Comment:
This looks like something we should strive for (though I guess for now it is better to keep it here :) )

Reply:
agree :)